benchmarks/ASQA/asqa_benchmark.py

```diff
@@ -57,8 +58,8 @@ def _evaluate(self, ) -> Tuple[Dict[Any, Any], Dataset]:
         results = {}
         for m in self.metrics:
             if m.name in ground_truths:
-                print(f"Evaluating {m.name}...")
-                self.prepare_data(*ground_truths[m.name])
+                print(f"Calculating {m.name}...")
+                self.prepare_data(m.name, *ground_truths[m.name])
                 results[m.name], self.dataset = m.compute(self.dataset, self.batch_size)
                 self.dataset = self.dataset.map(lambda example: {f"{m.name}.{ground_truths[m.name][0]}": ground_truths[m.name][1]}) # Add the ground truth column name
```

Comment:
Again, why not use rename?

Reply:
Because `gt_answer` may not exist before this step. And `prepare_data` function also do other things such as removing the metric_name column to avoid conflicts.